### DISCUSSION TITLE
Prevent specifics commands/action

### DISCUSSION THREAD

**Discussion by ceberterreche** [facebook/lexical#4486] [Q&A] [answered]
Hi,

I would like to only allow the user to create comments and prevent him from performing any other actions.

Is there a way to do this?

Thank you

**Comment by ivailop7** [comment, marked as answer]
You can make the editor read-only and disable any toolbar buttons you may have and have the popover on text selection show up to add a comment. The playground already offers the read-only example (lock icon, bottom right), you just have to edit the commenting plugin to ignore the read-only flag.
